src/pages/admin/reports/components/program-expenses-section/components/common/add-program-expense-record-modal/AddProgramExpenseRecordModal.module.scss

```
@@ -316,3 +316,25 @@
+.info-icon {
+    width: 18px;
+    height: 18px;
+
+    path {
+        stroke: c.$blue-500;
+    }
+}
+
+.info-text {
+    @include type.small-text-medium;
+    margin: 0;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)